apps/meteor/app/oembed/server/constants.ts

```diff
@@ -0,0 +1 @@
+export const MAX_EXTERNAL_URL_PREVIEWS = 5;
```

Comment:
btw, not sure this need to be a "global constant", since it is being used only in one file.

Reply:
I changed this to a global variable in the `server` file